src/app/server.js

```diff
@@ -52,7 +52,7 @@ app.use(session({
   saveUninitialized: false,
   store: memoryStore
 }))
-
+app.use(bodyParser.json())
```

Comment:
- Why this is introduced at the app level?

Reply:
This is i need to add the payload but there is no req.body. So i added this